image/templates/helm/stackrox-central/templates/02-config-controller-03-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+{{- include "srox.init" . -}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: config-controller
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: config-controller
+    {{- include "srox.labels" (list . "deployment" "config-controller") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "deployment" "config-controller") | nindent 4 }}
+spec:
+  replicas: 1
+  minReadySeconds: 15
+  selector:
+    matchLabels:
+      app: config-controller
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      labels:
+        app: config-controller
+        {{- include "srox.podLabels" (list . "deployment" "config-controller") | nindent 8 }}
+      annotations:
+        {{- include "srox.podAnnotations" (list . "deployment" "config-controller") | nindent 8 }}
+        {{- if eq ._rox.env.openshift 4 }}
+        openshift.io/required-scc: nonroot-v2
+        {{- end }}
+    spec:
+      serviceAccountName: config-controller
+      securityContext:
+        fsGroup: 4000
+        runAsUser: 4000
```

Comment:
Do we need these lines `fsGroup` and `runAsUser`?  I'd guess not. I think they are related to the need to use `nonroot-v2` SCC.

Reply:
I don't think they are related to scc only. I think fsGroup and runAsUser are good to keep in the security context.

Reply:
Can you explain why?  I think openshift handles these security details quite well by default, but perhaps there is a case for configuring them for non-openshift K8s distributions.

Reply:
Yes for the non openshift ones, that is why we have them on central from way back in the day (when we didn't support openshift)

Reply:
Is the goal to de-escalate privilege for the container by ensuring the user/group IDs are > 1000?

> Yes for the non openshift ones

In that case, can we only specify the `securityContext` in non-openshift installs?  There should be a variable available to toggle it. e.g. `{{- if not eq ._rox.env.openshift 4 }}`